Sage/carbon
# feat(tabs): describe tabs using design tokens, update tests

### Proposed behaviour

Describes tabs components using design tokens.
Describes validation-icon using design tokens.
Changes the height of the tab for a large size: `var(--sizing600)` -> `48px`.
Changes the shade of grey for the background in the hover state, the color of the unselected indigator and the left, right and top borders for the tabs: `var(--colorsActionMinor100)`.
Changes the outline border thickness for focus:  `var(--borderWidth300)` => `3px`.
Changes of padding value for `StyledTabsHeaderWrapper` related to change of border thickness for focus state: `3px`
Updates tests when the above changes are made.

### Current behaviour

Tabs and validation-icon use the theme styled-component property.
Currently the height for tabs for size M and L is the same and is `40px`.
Colors are currently used:
 - hover background: `#ccd6db`
 - border colors: `#ccd6db`
 - unselected indicator colors: `#ccd6db`

Current outline border thickness for focus - `2px`.
Current padding value for `StyledTabsHeaderWrapper` : `2px`

### Checklist




#### QA


### Additional context

Horizontal Tab: left side size M, right side size L
![image](https://user-images.githubusercontent.com/44143630/152367816-cfdf1180-dd40-4cfa-b946-22ec154c1f8a.png)

Vertical Tab: left side size M, right side size L
![image](https://user-images.githubusercontent.com/44143630/152368064-a30f7a10-3a61-4603-b905-28915e87d12d.png)

### Testing instructions

The correct appearance can be checked in the component: Tabs.
Use devtools to check if tabs use css variables.
